include/mbedtls/ssl.h

```diff
@@ -4854,14 +4854,15 @@ int mbedtls_ssl_get_session(const mbedtls_ssl_context *ssl,
  *                 DTLS records.
  *
  * \note           Defragmentation of incoming handshake messages in TLS
- *                 (excluding DTLS) is supported with some limitations:
+ *                 is supported with some limitations:
  *                 - On an Mbed TLS server that only accepts TLS 1.2,
  *                   the initial ClientHello message must not be fragmented.
  *                   A TLS 1.2 ClientHello may be fragmented if the server
  *                   also accepts TLS 1.3 connections (meaning
  *                   that #MBEDTLS_SSL_PROTO_TLS1_3 enabled, and the
  *                   accepted versions have not been restricted with
  *                   mbedtls_ssl_conf_max_tls_version() or the like).
+ *                   This limitation does not apply to DTLS.
```

Comment:
It's not just that limitation, the next one doesn't even make sense for DTLS (there is no concept of a "first" fragment) and neither does the one after (again, no ordering). This entire list is about TLS. Defragmentation for DTLS is entirely different and is fully supported, period. (It has been fully implemented from the start as opposed to added in a hurry as an afterthought...)

I stand by my earlier suggestion.